app/src/main/res/values/strings.xml

```diff
@@ -41,4 +46,20 @@
     <string name="shared_preferences_groups_key" translatable="false">user_groups</string>
     <string name="title_activity_record_item_list">RecordItemListActivity</string>
 
+    <!-- device location -->
+    <string name="location_fine_permission_rationale">Device location permission required. Records cannot be entered without a location.</string>
+    <string name="location_gps_needed_rationale">GPS must be enabled to get an accurate location for the record.</string>
+    <string name="location_requires_gps">Device does not have GPS, which is needed to get location. Please use a different device.</string>
+    <string name="location_gps_fix_found">GPS fix found. Getting location&#8230;</string>
+    <string name="location_best_found">Location found.</string>
+    <string name="location_awaiting_gps_fix">Waiting for GPS fix&#8230;</string>
+    <string name="location_service_off">Location check is not running.</string>
+    <string name="location_display" translatable="false">%1$.6f, %2$.6f</string>
+
+    <!-- location not set warning dialog -->
+    <string name="location_not_set_dialog_title">Location not set!</string>
+    <string name="location_not_set_dialog_warning">Record cannot be uploaded without a location. Continue to wait for GPS reading?</string>
```

Comment:
I found this a bit confusing when it popped up after a GPS fix had been received but before a best location had been estimated because it makes it seem as if no location data is available at all, but when you save it, it just uses the most recent fix location. It might be good to display a different message for the fix-but-no-location situation.